index.js

```diff
@@ -1 +1,64 @@
-module.exports = {};
+const fs = require('fs/promises');
+const { stripIndent } = require('common-tags');
+const { moduleTypes } = require('./config');
+const {
+  handleFieldType,
+  groupField,
+  writeFile
+} = require('./helpers');
+
+module.exports = {
+  construct(self, options) {
+    self.addTask(
+      'export',
+      stripIndent`
+        Exports A2 schemas to A3 format by outputting a "schema.js" file in every module folder.
+
+        Option:
+        * --folder: folder name relative to root where to search for modules. By default, it is "lib/modules". Usage: --folder=src/lib/modules
+      `,
+      async(apos, argv) => {
+        const { folder = 'lib/modules' } = argv;
+
+        for (const aposModule of Object.values(apos.modules)) {
+          const nativeModules = [ 'apostrophe-', '-auto-pages' ];
+          const isCustomModule = !nativeModules.some(nativeModule => {
+            return aposModule.__meta.name.includes(nativeModule);
+          });
+
+          if (!isCustomModule || !aposModule?.schema.length) {
+            continue;
+          }
```

Comment:
Why are custom `apostrophe-` modules excluded?

Reply:
The module is intended for custom modules